crates/core/c8y_api/src/smartrest/topic.rs

```diff
@@ -44,6 +44,10 @@ impl C8yTopic {
     pub fn accept(topic: &Topic) -> bool {
         topic.name.starts_with("c8y")
     }
+
+    pub fn accept_devicecontrol(topic: &Topic) -> bool {
```

Comment:
You may have to move this out of this module named `smartrest`. The `json_c8y` module might be a better fit.